configs/nimbus.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  description: Nimbus Feature Manifest for Python Testing
+channels:
+  - nightly
+  - developer
+features:
+  example-feature:
+    description: An example feature
+    variables:
+      enabled:
+        description: If the feature is enabled
+        type: Boolean
+        default: false
+      something:
+        description: Another variable
+        type: Option<String>
+        default: null
+    defaults:
+      - channel: nightly
+        value: { "enabled": true }
```

Comment:
The example here doesn't show it in quotes, but I presume it'll still work?
https://experimenter.info/fml-spec/#feature-defaults-and-channels

Reply:
I stole it from blurts